src/index.html

```diff
@@ -1,16 +1,62 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+    http-equiv="X-UA-Compatible"
+    content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+    rel="stylesheet"
+    href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+
+  <body class="page__body">
+
+    <div class="container">
+      <!-- #region formbig-->
+      <form
+      action="post"
+      class="form"
+      data-qa="big">
```

Comment:
When writing a multiline tag, the attributes should always be an indentation level ahead of the tag and it should be closed on a new line:

Good indentation example:
```html
<input
  type="text"
  name="surname"
  id="surname"
  required
>
```